.travis.yml

```diff
@@ -59,9 +59,10 @@ jobs:
               - docker run -u $UID --privileged -t --rm
                 --volume=/etc/passwd:/etc/passwd:ro
                 --volume=$PWD:/hlwm:rw
+                --volume=$HOME/.ccache-eoan-clang:/.ccache:rw
                 --volume=$HOME/.tox-cache:/hlwm/.tox:rw
                 eoan sh -c '
-                    /hlwm/ci/build.py --build-dir=/hlwm/build-clang-9 --cxx=clang++-9 --cc=clang-9 --build-type=Debug --check-using-std --iwyu --clang-tidy --flake8
```

Comment:
Now that --clang-tidy is not used here anymore, shall we remove it?
Or keep it for now (for local use maybe?)

Reply:
Yeah, I'd rather keep it, might be useful.

Reply:
Ok!

Reply:
Now I had to remove it after all, because I didn't want to have the .clang-tidy lie around...